api/conftest.py

```diff
@@ -58,6 +58,10 @@
 from projects.tags.models import Tag
 from segments.models import Condition, Segment, SegmentRule
 from task_processor.task_run_method import TaskRunMethod
+from tests.types import (
+    WithEnvironmentPermissionsCallable,
+    WithProjectPermissionsCallable,
+)
```

Comment:
I love these types, but can your PR include them in all present test function types? It should be an easy replace and then they'd be fixed everywhere.

Reply:
Yep, on it.